### PR TITLE
Log airflow webserver to stderr

### PR DESCRIPTION
**Pull request by lawliet89**
Also bind to `0.0.0.0`

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ if [ "$1" = 'afp-scheduler' ]; then
 elif [ "$1" = 'afp-webserver' ]; then
   echo "Starting webserver"
   python "${AIRFLOW_HOME}"/setup_auth.py
-  exec gosu "${USER}" airflow webserver
+  exec gosu "${USER}" airflow webserver --log-file=- --hostname=0.0.0.0
```
Comment (chrissng) [inline entrypoint.sh]: what does `--log-file=-` means?

Comment (lawliet89) [reply entrypoint.sh]: Log to `stderr`

See https://airflow.incubator.apache.org/cli.html

File: entrypoint.sh [hunk 2]
```diff
@@ -49,7 +49,7 @@ if [ "$1" = 'afp-scheduler' ]; then
 elif [ "$1" = 'afp-webserver' ]; then
   echo "Starting webserver"
   python "${AIRFLOW_HOME}"/setup_auth.py
-  exec gosu "${USER}" airflow webserver
+  exec gosu "${USER}" airflow webserver --hostname=0.0.0.0 --log-file=- --debug
```
Comment (chrissng) [inline entrypoint.sh:52]: Hmm don't quite understand why these flags are needed

**Comment by lawliet89** [comment]
Debug is to log access. The host name is not needed.

Actually can close the pr because it's not needed at the moment.

**Comment by lawliet89** [comment]
Actually this is not needed for now. I'll close this for now.

**Pull request closed without merging**